RealestateApi/Dto/Request/CustomDtoRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace RealEstateApi.Dto.Request
+{
+    public class CustomDtoRequest
```

Comment:
Class name should change to CustomerRequestDto

Reply:
Done!